static/src/javascripts/projects/commercial/modules/dfp/prepare-sonobi-tag.js

```diff
@@ -0,0 +1,24 @@
+define([
+    'common/modules/commercial/commercial-features',
+    'commercial/modules/dfp/dfp-env'
+], function(
+    commercialFeatures,
+    dfpEnv
+){
+    var prom = null;
```

Comment:
Cool, very tidy. I guess `require` is memoized, in a sense. what i found was i could memoize `setupSonobi`, but if i memoized `init`, the second call still executed. do you know why?